matchify-backend/.env

```
@@ -0,0 +1,2 @@
+MONGO_URI=mongodb://localhost:27017
+MONGO_DB=matchify
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)